sphinxcontrib/openapi/renderers/_httpdomain.py

```diff
@@ -1,10 +1,9 @@
 """OpenAPI spec renderer."""
 
 import sys
-if sys.version_info >= (3, 3):
```

Comment:
Don't we need to keep the guard? I don't remember for sure but thought that `collections.abc` was added in 3.3

Reply:
`setup.py` already sets `python_requires=">=3.6"`, so we don't need to maintain old compat cold for older versions.

Reply:
Ah right, I missed that point.